core-java-modules/core-java-collections-set-2/src/main/java/com/baeldung/cartesianproduct/CartesianProduct.java

```diff
@@ -0,0 +1,81 @@
+package com.baeldung.cartesianproduct;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.google.common.collect.Sets;
+
+public class CartesianProduct {
+    // Responsible for calculating Cartesian Product using iterative approach.
```

Comment:
let's remove the comments and make sure the method naming reflects its intent/objective

Reply:
Done